engine/runtime/src/main/java/org/enso/interpreter/runtime/ResourceManager.java

```diff
@@ -186,6 +204,21 @@ private synchronized void removeFromItems(PhantomReference<ManagedResource> it)
     }
   }
 
+  /**
+   * Explicitly schedules all the <em>system references</em> registered with the manager for
+   * finalization.
+   *
+   * @see #register
+   */
+  @CompilerDirectives.TruffleBoundary
+  public final synchronized void scheduleFinalization() {
```

Comment:
nitpick, but IMO `scheduleFinalizationOfSystemResources` could be better
```suggestion
  public final synchronized void scheduleFinalizationOfSystemResources() {
```

The name `scheduleFinalization` sounds like it may finalize _all_ references.

Documentation clarifies this, so I'm ok to keep it as is though.

Reply:
Renamed in 068d223b1a